### PR TITLE
`Typed` tests

### PR DESCRIPTION
**Pull request by 1Shubham7**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/kubeclientbridge/clientset_bridge_test.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import (
+    "testing"
+
+    "github.com/kubeedge/kubeedge/edge/pkg/metamanager/client"
+    kecorev1 "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge/typed/core/v1"
+    "github.com/stretchr/testify/assert"
+    fakekube "k8s.io/client-go/kubernetes/fake"
+
+	kestoragev1 "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge/typed/storage/v1"
+	kecoordinationv1 "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge/typed/coordination/v1"
```
Comment (Shelley-BaoYue) [inline edge/pkg/edged/kubeclientbridge/clientset_bridge_test.go:28]: Please fix the wrong format : )

**Review by Shelley-BaoYue — COMMENTED** [review]
/kind test
/lgtm

**Comment by 1Shubham7** [comment]
@WillardHu please review this one as well :)

**Pull request merged**